composer.json

```diff
@@ -21,13 +21,13 @@
     "extra": {
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ^8.0",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
         "laminas/laminas-coding-standard": "~1.0.0",
         "laminas/laminas-servicemanager": "^2.7.5 || ^3.0.3",
```

Comment:
Version 2 of laminas-servicemanager will no longer work with the newer PHP versions.

Reply:
I believe composer will handle this and use v3 with php8. Version 2 of service manager should still work with php7.3 and php7.4. Or have I missed something?

Reply:
Sorry, yes, it works, but the old version can be dropped here.

Reply:
if I drop it, won't it potentially break code in projects using this package unnecessarily?

Reply:
Version 3 is still there and if someone explicitly relies on version 2, the newer version of this package would not be installed.

Reply:
I feel this change is beyond the original scope so unless it is going to prevent acceptance of the PR I'd prefer to leave the change to another PR.

Reply:
@far-blue
Please remove version 2 of laminas-servicemanager, it cannot be installed with the specified PHP versions.

Reply:
I feel this change is beyond the original scope so unless it is going to prevent acceptance of the PR I'd prefer to leave the change to another PR.